app/src/main/java/com/github/fribourgsdp/radio/User.kt

```diff
@@ -46,10 +42,14 @@ data class User (var name: String, val color: Int) {
 
 
     companion object {
+        const val USER_DATA_PATH = "user_data_file"
+        private var fileSystemGetter: FileSystem.FileSystemGetter =
```

Comment:
If I understand correctly, this is the working FS implementation we previously used wrapped in a a class, such that you can mock it when needed in MockFileSystem right?
Even though it's not the most "logical" place to put this attribute considering a User shouldn't store the info that indicates where it is going to be stored, I understand it's by far the easiest way to do so, so it's fine for me.

Reply:
Yes. I am using a getter instead of the actual filesystem directly to preserve backwards compatibility with the context signature. I am forced to put it in the companion object instead of a class instance because it also needs to be used by the static load method. The USER_DATA_PATH variable has just been moved from a global variable to the companion object because I find it cleaner. We use this path because it is the default path for user files, but you could technically store several users if you choose several paths.